packages/react/src/components/Typography/Paragraph/Paragraph.tsx

```diff
@@ -45,12 +39,11 @@ export const Paragraph = forwardRef<HTMLParagraphElement, ParagraphProps>(
       <Component
         ref={ref}
         className={cl(
-          'ds-paragraph',
+          variant === 'default' ? 'ds-paragraph' : `ds-paragraph-${variant}`,
```

Comment:
skulle `variant` også vært `data-variant`, eller skulle den hete noe mer beskrivende - for det påvirker kun line-height? `data-line-height`? 😇

Reply:
We'll have to take up with the rest of the team so it matches Figma and elsewhere :)